routes/user.js

```diff
@@ -25,64 +25,66 @@ router.get("/api/user", (req, res) => {
     ],
     affiliation: { $regex: affiliation, $options: "i" },
   };
-  users
-    .find(query)
-    .then((users) => {
-      returnData(
-        users.map((user) => ({
-          _id: user._id,
-          name: user.name,
-          email: user.email,
-          affiliation: user.affiliation,
-          school: user.school,
-          grade: user.grade,
-          whitelisted_plan_ids: user.whitelisted_plan_ids,
-        })),
-        res
-      );
-    })
-    .catch((err) => errorHandler(res, 400, err));
+  try {
+    let users = await Users.find(query).exec();
+    users = users.map((user) => ({
+      _id: user._id,
+      name: user.name,
+      email: user.email,
+      affiliation: user.affiliation,
+      school: user.school,
+      grade: user.grade,
+      whitelisted_plan_ids: user.whitelisted_plan_ids,
+    }));
+    returnData(users, res);
+  } catch (err) {
+    errorHandler(res, 400, err);
+  }
 });
 
 // TODO: Temporarily disabled for OOSE dev deployment
 // if (DEBUG) {
```

Comment:
We can probably uncomment this now... I don't see any big issues with uncommenting this and it's good.